src/components/dialog/dialog.js

```diff
@@ -503,7 +504,7 @@ function MdDialogProvider($$interimElementProvider) {
       options: advancedDialogOptions
     })
     .addPreset('prompt', {
-      methods: ['title', 'htmlContent', 'textContent', 'content', 'placeholder', 'ariaLabel',
+      methods: ['title', 'htmlContent', 'textContent', 'defaultValue', 'content', 'placeholder', 'ariaLabel',
```

Comment:
I respected the `window.prompt` reference here.
- https://developer.mozilla.org/de/docs/Web/API/window/prompt